Dockerfile

```diff
@@ -5,6 +5,7 @@ FROM openjdk:15-alpine3.11
 WORKDIR /usr/app
 COPY . ./
 RUN apk add --update \
+    yarn \
```

Comment:
Here we're sneaking in a small change to switch our package manager from npm to yarn.  Yes, yes I'm totally pork-barreling here.